apps/docs/src/css/custom.css

```
@@ -1,8 +1,7 @@
+@import url(https://api.fontshare.com/v2/css?f[]=work-sans@2,1&display=swap);
```

Stylelint (17.14.0)

[error] Expected "url(https://api.fontshare.com/v2/css?f[]=work-sans@2,1&display=swap)" to be ""https://api.fontshare.com/v2/css?f[]=work-sans@2,1&display=swap"" (import-notation)

(import-notation)